static/app/components/feedback/feedbackSearch.tsx

```diff
@@ -1,31 +1,116 @@
-import {CSSProperties} from 'react';
+import {CSSProperties, useCallback} from 'react';
 import {browserHistory} from 'react-router';
 import styled from '@emotion/styled';
 
+import {fetchTagValues} from 'sentry/actionCreators/tags';
+import SmartSearchBar from 'sentry/components/smartSearchBar';
 import {t} from 'sentry/locale';
+import {Tag, TagCollection, TagValue} from 'sentry/types';
+import {isAggregateField} from 'sentry/utils/discover/fields';
+import {
+  FEEDBACK_FIELDS,
+  FeedbackFieldKey,
+  FieldKey,
+  FieldKind,
+  getFieldDefinition,
+} from 'sentry/utils/fields';
 import {decodeScalar} from 'sentry/utils/queryString';
+import useApi from 'sentry/utils/useApi';
 import {useLocation} from 'sentry/utils/useLocation';
 import useOrganization from 'sentry/utils/useOrganization';
 import usePageFilters from 'sentry/utils/usePageFilters';
-import ReplaySearchBar from 'sentry/views/replays/list/replaySearchBar';
+import useTags from 'sentry/utils/useTags';
+
+const EXCLUDED_TAGS = [
+  FeedbackFieldKey.BROWSER_VERSION,
+  FeedbackFieldKey.EMAIL,
+  FeedbackFieldKey.LOCALE_LANG,
+  FeedbackFieldKey.LOCALE_TIMEZONE,
+  FeedbackFieldKey.MESSAGE,
+  FeedbackFieldKey.NAME,
+  FieldKey.PLATFORM,
+  FeedbackFieldKey.OS_VERSION,
+];
```

Comment:
Are these excluded because they are not yet working?

Reply:
yes